es6-classes/square.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+
+class Square extends Shape {
+  constructor(width) {
+    super(width);
+    this.width = width;
+    this.area = width * width;
+    this.perimeter = width * 4;
```

Comment:
Instead of setting these properties manually, we should take advantage of the `super` keyword. This allows you to call methods on the parent class.